sdk/tests/client/input_selection/account_outputs.rs

```diff
@@ -2071,3 +2071,62 @@ fn min_allot_account_mana_requirement_covered_2() {
     );
     assert_eq!(selected.transaction.outputs()[1].as_account().mana(), 0);
 }
+
+#[test]
+fn implicit_account_transition() {
+    let protocol_parameters = protocol_parameters();
+    let account_id_1 = AccountId::from_str(ACCOUNT_ID_1).unwrap();
+
+    let inputs = [BasicOutputBuilder::new_with_amount(1_000_000)
+        .add_unlock_condition(AddressUnlockCondition::new(Address::ImplicitAccountCreation(
+            ImplicitAccountCreationAddress::new(
+                **Address::try_from_bech32(BECH32_ADDRESS_ED25519_0).unwrap().as_ed25519(),
+            ),
+        )))
+        .with_mana(961)
+        .finish_output()
+        .unwrap()];
+    let inputs = inputs
+        .into_iter()
+        .map(|input| InputSigningData {
+            output: input,
+            output_metadata: rand_output_metadata_with_id(rand_output_id_with_slot_index(SLOT_INDEX)),
+            chain: None,
+        })
+        .collect::<Vec<_>>();
+
+    let input_output_id = *inputs[0].output_id();
+    let account_id = AccountId::from(&input_output_id);
+    let outputs = vec![
+        AccountOutputBuilder::new_with_amount(1_000_000, account_id)
+            .add_unlock_condition(AddressUnlockCondition::new(
+                Address::try_from_bech32(BECH32_ADDRESS_ED25519_0).unwrap(),
+            ))
```

Comment:
We had a call with Chloé today where we said we would add semantic validation inside ISA. When we do that, this test will fail because the account should have a block issuer feature.

Reply:
But I'm fine with fixing the test when it starts failing